docs/developer-experience/artificial-intelligence.md

```diff
@@ -0,0 +1,63 @@
+
+Developers on our team have found the following use-cases for GitHub Copilot:
+
+- __Write Documentation__. For example, the above paragraph was written using Copilot.
+- __Write Unit Tests__. Given that setup and assertions are often consistent across unit tests, Copilot tends to be very accurate.
+- __Unblock__. It is often hard start writing when staring at a blank page, Copilot can fill the space with something that may or may not be what you ultimately want to do, but it can help get you in the right headspace.
+
+If you want Copilot to write something useful for you, try writing a comment that describes what your code is going to do - it can often take it from there.
+
+## GitHub Copilot Labs
+
+Copilot has extension that offers additional features that are not yet ready for prime-time. You can install it from the VS Code Marketplace. These features include:
+
+- __Explain__. Copilot can explain what the code is doing in natural language.
+- __Translate__. Copilot can translate code from one language to another.
+- __Brushes__. You can select code that Copilot then modifies inline based on a "brush" you select, for example, to make the code more readable, fix bugs, improve debugging, document, etc.
+- __Generate Tests__. Copilot can generate unit tests for your code. Though currently this is limited to JavaScript and TypeScript.
+
+## GitHub Copilot X
+
+This extention for Copilot is currently in technical preview. It will enable the following additional features:
+
+- __Chat__. Rather than just providing code completion, Copilot will be able to have a conversation with you about what you want to do. It has context about the code you are working on and can provide suggestions based on that context.
+- __Explain__. Copilot can explain what the code is doing in natural language.
+- __Write Code__. Given prompting by the developer it can write code that you can one-click deploy into existing or new files.
+- __Debug__. Copilot can analyze your code and propose solutions to fix bugs.
+
+It can do most of what Labs can do with "brushes" as "topics", but whereas Labs changes the code in your file, the chat functionality just shows what it would change in the window. However, there is also an "inline mode" for GitHub Copilot Chat that allows you to make changes to your code inline which does not have this same limitation.
+
+You must sign-up for the preview [here](https://github.com/features/preview/copilot-x). Once you are accepted, you will be able to install the extension from the VS Code Marketplace. You may require the Insiders build of VS Code.
+
+## ChatGPT / Bing Chat
```

Comment:
I would consider focusing this page on copilots and create another content for ChatGPT/Bing Chat.

Reply:
I agree on focusing the page on copilots... 
Regarding placing ChatGPT/Bing Chat elsewhere, @shiranr do you feel that content fits in the EF playbook, or did you mean perhaps elsewhere as in not in the playbook?

Reply:
This specific content is not relevant to the EF playbook but if people are using ChatGPT / Bing Chat for programming and can give a solid to do/ not to do about it, it MIGHT be good content for the playbook